src/AutoRest.CSharp/Common/Input/CodeModelInputs.cs

```diff
@@ -6,5 +6,5 @@
 #pragma warning disable SA1649
 namespace AutoRest.CSharp.Common.Input
 {
-    internal record CodeModelType(Schema Schema, bool IsNullable) : InputType(Schema.Name, IsNullable);
+    internal record CodeModelType(Schema Schema) : InputType(Schema.Name);
```

Comment:
I wonder why we still have this type.
Should not we already clean up all the usage of this when we were doing the consolidation for MPG with input types?
@live1206

Reply:
This should be removed, updated it in https://github.com/Azure/autorest.csharp/pull/4834

Reply:
removed

Reply:
Please rmeove this type. This has been removed in a previous PR, and it should not be here.